tests/cli/test_rasa_train.py

```diff
@@ -64,6 +64,43 @@ def test_train(run_in_simple_project: Callable[..., RunResult], tmp_path: Path):
     )
 
 
+def test_train_with_endpoints(
+    run_in_simple_project: Callable[..., RunResult], tmp_path: Path
+):
+    temp_dir = os.getcwd()
+
+    run_in_simple_project(
+        "train",
+        "-c",
+        "config.yml",
+        "-d",
+        "domain.yml",
+        "--data",
+        "data",
+        "--out",
+        "train_models",
+        "--fixed-model-name",
+        "test-model",
+        "--endpoints",
+        "endpoints.yml",
```

Comment:
Should we parameterize this when endpoints are given as argument, and when endpoints are not given as input to the command line and then the default file is read? Also is there a way to assert that the contents of `endpoints.yml` is correctly processed during training?

Reply:
Yes to the parametrization, good point.

Re. processing: It's not processed at all during training at the moment. If there was a reason to do so in the future, we could add it.